Tests/UnitTests/CardSets/Expert1CardsGenTests.cpp

```diff
@@ -8402,6 +8402,50 @@ TEST(NeutralExpert1Test, NEW1_021_Doomsayer)
     EXPECT_EQ(opField.GetCount(), 0);
 }
 
+// --------------------------------------- MINION - NEUTRAL
+// [NEW1_024] Captain Greenskin - COST:5 [ATK:5/HP:4]
+// - Race: Pirate, Faction: Neutral, Set: Expert1, Rarity: legendary
+// --------------------------------------------------------
+// "Text : "<b>Battlecry:</b> Give your weapon +1/+1.",
+// --------------------------------------------------------
+TEST(NeutralExpert1Test, NEW1_024_CaptainGreenskin)
+{
+    GameConfig config;
+    config.player1Class = CardClass::ROGUE;
+    config.player2Class = CardClass::MAGE;
+    config.startPlayer = PlayerType::PLAYER1;
+    config.doFillDecks = true;
+    config.autoRun = false;
+
+    Game game(config);
+    game.StartGame();
+    game.ProcessUntil(Step::MAIN_START);
+
+    Player& curPlayer = game.GetCurrentPlayer();
+    Player& opPlayer = game.GetOpponentPlayer();
+    curPlayer.SetTotalMana(10);
+    curPlayer.SetUsedMana(0);
+    opPlayer.SetTotalMana(10);
+    opPlayer.SetUsedMana(0);
+
+	const auto card1 = Generic::DrawCard(
+        curPlayer, Cards::GetInstance().FindCardByName("Captain Greenskin"));
+
+    game.Process(curPlayer, HeroPowerTask());
+    EXPECT_EQ(curPlayer.GetHero()->weapon->GetAttack(), 1);
+    EXPECT_EQ(curPlayer.GetHero()->weapon->GetDurability(), 2);
+
+    game.Process(curPlayer, PlayCardTask::Spell(card1));
```

Comment:
`Caption Greenskin` is not spell, but minion.

Reply:
fixed.